src/notebooks/record_dedupe.py

```diff
@@ -0,0 +1,111 @@
+from typing import Callable
+import pandas as pd
+
+PATH_TO_DATA = '/Users/Brandon/Downloads/Archive/Patient-1.ndjson'
+
+def make_test_dedupe_data(df: pd.DataFrame):
+    '''
+    Turn a bundle of patient data into a small test set for verifying
+    efficacy of deduplication. There will be 6 unique records here,
+    4 of which are completely dissimilar, one which will have three
+    duplicated versions of varying changes, and one which will be a 
+    near match for the duplicates but is off by one character.
+    '''
+    
+    # Full match case: everything the same, should be no merge conflicts
+    df.loc[6, :] = df.loc[0, :]
```

Comment:
I'm really rusty with pandas - why isn't it `df.loc[6] = df.loc[0]`? Would that do something entirely different?

Reply:
Sure. In my experience, it's best to be as absolutely precise as possible when working with `.loc`. One of the things I hate the most (and one of the most insidious ways a system can break) is when  your syntax causes pandas to hand you a `view` or a `reference` into a dataframe rather than the indexed data itself, or a `Series` instead of a partial dataframe. Specifying the nth row and explicitly asking for the _data_ in all columns ensures this is robust to actually fetching data and not a view. Granted, it's in a function that makes dummy test data and so doesn't matter with prod, but still, it's an important practice in my eyes (even if just so I don't slip on other stuff).